File: server/getScore.py
#! C:\Users\TJ\AppData\Local\Microsoft\WindowsApps\python.exe
print("Content-type: json\n\n")
import mysql.connector
mydb = mysql.connector.connect(
  host="localhost",
  user="root",
  password="",
  database="sci_smash"
)
myCursor = mydb.cursor()
myCursor.execute('SELECT score, username FROM scores ORDER BY score desc LIMIT 50')
result = myCursor.fetchall()

endstring = ""
endstring += "{\"items\": ["
for x in range(len(result)):
  y = result[x]
  endstring += "{ \"score\": "+str(y[0])+", \"username\": \""+y[1]+"\" }"
  if  x != len(result)-1:
    endstring += ","
  
endstring += "]}"
print(endstring)

File: server/getScore.php
<?php
$servername = "localhost";
$username = "root";
$password = "";
$dbname = "sci_smash";

// Create connection
$conn = new mysqli($servername, $username, $password, $dbname);

// Check connection
if ($conn->connect_error) {
  die("Connection failed: " . $conn->connect_error);
}
$sql = "SELECT score_id, score, username FROM scores ORDER BY score desc";
$result = $conn->query($sql);
$currentRank = 1;
if ($result->num_rows > 0) {
  // output data of each row
  while($row = $result->fetch_assoc()) {
    echo $currentRank . ": " . $row["username"]. " - " . $row["score"]. "\n";
    $currentRank = $currentRank + 1;
  }
} else {
  echo "0 results";
}
$conn->close();
?>